security-command-center/snippets/src/test/java/vtwo/IamIT.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo;
```

Comment:
why is this in the different package?

Reply:
These samples are part of the v2 api for security center.
https://cloud.google.com/security-command-center/docs/samples/securitycenter-test-iam?hl=en#securitycenter_test_iam-java

Reply:
I understand that. I think we do not have a particular guidelines about those. Usually it is more convenient to place the samples in the same namespace/package as the code being tested.